easybuild/easyconfigs/s/Schrodinger/Schrodinger_2014-1_Linux-x86_64.eb

```diff
@@ -0,0 +1,46 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock='Binary'
+
+name = "Schrodinger"
+version = "2014-1"
+versionsuffix = "_Linux-x86_64"
+
+homepage = 'http://www.schrodinger.com/'
+description = """ Schrödinger aims to provide integrated software solutions and services 
+ that truly meet its customers’ needs. We want to empower researchers around the world to 
+ achieve their goals of improving human health and quality of life through advanced computational 
+ techniques that transform the way chemists design compounds and materials. """
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = ['%(name)s_Suites_%(version)s%(versionsuffix)s.tar']
+
+# Usage:  INSTALL [<options> ] [<install-files>]
+#Options: 
+#  -d <distribution-dir>... directory containing the install files
+#  -m ..................... manual mode (no automatically included modules)
+#  -b ..................... batch install
+#  -s <SCHRODINGER> ....... installation directory 
+#  -k <tmpdir> ............ scratch directory
+#  -t <thirdparty-dir> .... install directory for third-party modules
+install_cmd = 'tar xfv Schrodinger_Suites_%(version)s%(versionsuffix)s.tar && '
```

Comment:
the above line can be merged to sources step, if so desired. 
IMHO, this may give more freedom in relation to paths/locations of files etc...